numpy/lib/_arraysetops_impl.py

```diff
@@ -430,7 +426,6 @@ def unique_all(x):
     -------
     out : namedtuple
         The result containing:
-
```

Comment:
This line break is necessary for the bulleted list formatting and should be added back.